thread-racing/scripts/embedded-broker.cli

```diff
@@ -0,0 +1,14 @@
+if (outcome == success) of /subsystem=messaging-activemq:read-resource
+  /subsystem=messaging-activemq/server=default:add(elytron-domain=ApplicationDomain)
+end-if
+if (outcome == success) of /subsystem=messaging-activemq/server=default:read-resource
+    /subsystem=messaging-activemq/server=default/in-vm-acceptor=in-vm:add(server-id=0, params=[buffer-pooling=false])
+    /subsystem=messaging-activemq/server=default/in-vm-connector=in-vm:add(server-id=0, params=[buffer-pooling=false])
+    /subsystem=messaging-activemq/server=default/pooled-connection-factory=activemq-ra:add(connectors=[in-vm], transaction=xa, entries=[java:/JmsXA,java:jboss/DefaultJMSConnectionFactory])
+    /subsystem=messaging-activemq/server=default/jms-queue=DLQ:add(entries=[java:/jms/queue/DLQ])
```

Comment:
the `helloworld-mdb` does not specify DLQ and Expiry Queue. Should we add them there too?